### PR TITLE
Fallback labels to MD5 if too long

### PR DESCRIPTION
**Pull request by andreaTP**
Resolves: #100

### CODE REVIEW

**Comment by vsevel** [comment]
interesting. does it change the `artifactId` param in the rest api? or the impact is only on this particular label?

**Comment by andreaTP** [comment]
> the impact is only on this particular label

this is correct. I included a super-long name in the E2E tests to verify the behavior and, as far as I can see, everything works correctly.

Please note that this change is also backward compatible 🙂

**Review by EricWittmann — APPROVED** [review]
lgtm

File: e2e/src/test/java/io/apicurio/e2e/OperatorTest.java [hunk 1]
```diff
@@ -39,6 +39,9 @@ public class OperatorTest {
 
     Logger logger = LoggerFactory.getLogger(getClass());
 
+    private static String TEST_ARTIFACT_ID =
+            "She-sells-sea-shells-by-the-sea-shore_The-shells-she-sells-are-surely-sea-shells_So-if-she-sells-shells-on-the-sea-shore_She-sells-seashore-shells";
```
Comment (EricWittmann) [inline e2e/src/test/java/io/apicurio/e2e/OperatorTest.java:43]: :)

**Pull request merged**